util/tracer_nvbit/tracer_tool/tracer_tool.cu

```diff
@@ -116,6 +118,8 @@ void nvbit_at_init() {
   GET_VAR_INT(xz_compress_trace, "TRACE_FILE_COMPRESS", 1,
               "Create xz-compressed trace"
               "file");
+  GET_VAR_INT(multi_process, "TRACE_MULTI_PROCESS", 0,
```

Comment:
Do we need a flag for this?
Can we just not have the default do multi-process with just a process of 1?